src/components/OrgPostCard/OrgPostCard.spec.tsx

```
@@ -1,980 +1,710 @@
+      'talawa-admin.org',
+    ];
+
+    // Check if hostname exactly matches or is subdomain of allowed domain
+    const isAllowedDomain = allowedDomains.some((domain) => {
+      // Exact match
+      if (parsed.hostname === domain) {
+        return true;
+      }
+
+      // Subdomain check - ensure it's actually a subdomain with proper structure
+      if (parsed.hostname.endsWith('.' + domain)) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)